bucket/ytmdesktop.json

```diff
@@ -0,0 +1,36 @@
+{
+    "version": "1.12.1",
+    "description": "A Desktop App for YouTube Music",
```

Comment:
I am missing clarification that this is not official product from Google.

Add something like `unofficial`, `third-party`, ...

Reply:
Agreed, I just grabbed that description from their GitHub page.